### PR TITLE
Prevent controls disappearing for ads flag

### PR DESCRIPTION
**Pull request by radium-v**
### This PR will...
Prevent the entire controls container from disappearing when in certain UI states.

#### Addresses Issue(s):
JW8-95

### CODE REVIEW

File: src/css/controls/imports/jwplayerlayout.less [hunk 1]
```diff
@@ -17,7 +17,7 @@
     transition: opacity 250ms cubic-bezier(0, 0.25, 0.25, 1);
     z-index: 0;
 
-    .jwplayer.jw-state-playing.jw-flag-user-inactive:not(.jw-flag-casting) & {
+    .jwplayer.jw-state-playing.jw-flag-user-inactive:not(.jw-flag-casting):not(.jw-flag-ads) & {
```
Comment (egreaves) [inline src/css/controls/imports/jwplayerlayout.less]: We should look into reducing the specificity by removing `.jwplayer`. 

Also, though the skip button should be visible, the other controls aren't visible during ad playback when `.jw-flag-user-inactive` is present.

Comment (radium-v) [inline src/css/controls/imports/jwplayerlayout.less]: The `.jw-controlbar` element has its own transition state that's triggered when the `.jw-flag-user-inactive` is activated.

**Comment by hghazzi** [comment]
test this please

**Comment by egreaves** [comment]
test this please

File: src/css/controls/imports/jwplayerlayout.less (hunk 1 above)
Comment (egreaves) [inline src/css/controls/imports/jwplayerlayout.less]: Pulled it down and tested it. You're right.

**Pull request merged**